### ISSUE TITLE
The Proposed Recommendation phase is _not_ eliminated

### ISSUE THREAD

**Issue by nigelmegitt** [w3c/w3process#510]
Editorial suggestion:

§6.4.2 says:

>The Proposed Recommendation phase is eliminated. Instead, an Advisory Committee Review is started upon publication of each Candidate Registry Snapshot. 

This could be misinterpreted as applying to the actual Proposed Recommendation phase. Suggest rewording to:

>There is no equivalent to the Proposed Recommendation phase for Registries.  Instead, an Advisory Committee Review is started upon publication of each Candidate Registry Snapshot.

**Comment by frivoal** [comment]
Thanks. I don't think the original was ambiguous, given that the context is listing differences from the REC track, but even then it could be misleading if you're not paying attention to the context, and the new phrasing is just all around better. Taking it in as a small editorial tweak.
